test/test_rules.py

```diff
@@ -0,0 +1,2331 @@
+import pytest
+
+from pyseeyou.locales import get_parts_of_num
+
+# ========================
+# GENERATED AUTOMATICALLY
+# DON'T MODIFY MANUALLY
+# ========================
+
+from pyseeyou.cldr_rules import (
+    cardinal_af, cardinal_ak, cardinal_am, cardinal_ar, cardinal_ast, cardinal_be, cardinal_bm, cardinal_br, cardinal_bs, cardinal_cs, cardinal_cy, cardinal_da, cardinal_dsb, cardinal_ff, cardinal_fil, cardinal_ga, cardinal_gd, cardinal_gv, cardinal_he, cardinal_is, cardinal_iu, cardinal_ksh, cardinal_lag, cardinal_lt, cardinal_lv, cardinal_mk, cardinal_mo, cardinal_mt, cardinal_pl, cardinal_pt, cardinal_ru, cardinal_shi, cardinal_si, cardinal_sl, cardinal_tzm)
+
+def test_cardinal_af():
+    assertions = [
+        ('one', ['1','1.0','1.00','1.000']),
+        ('other', ['0','2','9','16','100','0.0','0.45','0.9','1.1','1.35','1.6','10.0'])
+    ]
+
+    for assertion in assertions:

```

Comment:
It seems it is possible to extract this code to a method and so to avoid code duplication. I look into that later.

```py
def check(assertions, plural_func):
    for assertion in assertions:
        match, samples = assertion
        for sample in samples:
                assert plural_func(*get_parts_of_num(sample)) == match

def test_cardinal_af():
    assertions = [
        ('one', ['1','1.0','1.00','1.000']),
        ('other', ['0','2','9','16','100','0.0','0.45','0.9','1.1','1.35','1.6','10.0'])
    ]

    check(assertions, cardinal_af)
```

Reply:
I think that'd be a great change to make now. This test file is quite large with a lot of repeated code, so this could be an easy win for cutting that down. 

Plus, I think I've figured out why CircleCI wasn't building for this PR, and all it should need is a new commit to trigger 👍 .